pkg/localize/locales/en/cmd/kafka.en.toml

```diff
@@ -299,7 +299,7 @@ description = 'Help text for Cloud Region'
 one = "Geographical region where the Kafka instance will be deployed"
 
 [kafka.create.argument.name.error.requiredWhenNonInteractive]
-one = 'name is required. Run "rhoas kafka create --name <name>"'
+one = 'name is required. Run "rhoas kafka create --name name"'
```

Comment:
```suggestion
one = 'name flag is required."'
```

Reply:
`<name>` was used as a placeholder value. If we need to remove that we should use a more realistic instance name instead of just "name".

Reply:
I think this is little bit redundant but please ignore my comment and use @craicoverflow suggestion.

`1iSY6RQ3JKI8Q0OTmjQFd3ocFRg` is example id